migration-util/src/main/java/edu/unc/lib/dcr/migration/premis/ContentPremisToRdfTransformer.java

```diff
@@ -307,7 +308,11 @@ private void addFixityEvent(Element eventEl, String eventOutcome) {
     }
 
     private void addEvent(Element eventEl, Resource eventType, String eventDetail, SoftwareAgent agent) {
-        addEvent(eventEl, eventType, eventDetail, true, agent.getFullname());
+        String agentFullname = agent.getFullname();
```

Comment:
i'm guessing these changes won't be necessary after updating depositBxc3 to have the name of the service, so you should be able to roll them back